#!/usr/bin/env python
PACKAGE = "cloud_generator"
import roslib;roslib.load_manifest(PACKAGE)
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("gpuCloud",   bool_t,   0, "gpuCloud",  True)
gen.add("gpuEdge",   bool_t,   0, "gpuEdge",  True)
gen.add("xRange", double_t, 0, "xRange",   3., 0., 8.)
gen.add("yRange", double_t, 0, "yRange",   3., 0., 8.)
gen.add("zRange", double_t, 0, "zRange",   0.0, 0., 8.)
gen.add("xShift", double_t, 0, "xShift",   0.0, -0.05, 0.05)
gen.add("yShift", double_t, 0, "yShift",   0.0, -0.05, 0.05)
gen.add("zShift", double_t, 0, "zShift",   0.0, -0.05, 0.05)
exit(gen.generate(PACKAGE, "cloud_generator", "cloud_generator"))
